jenkins/shared-library/vars/publish.groovy

```diff
@@ -30,7 +30,9 @@ void dockerPublisher(Map config, String phase) {
 	final String imageVersionName = "${config.dockerImageName}:${version}"
 	final String archImageName = imageVersionName + "-${ARCHITECTURE}"
 	dockerHelper.loginAndRunCommand(DOCKER_CREDENTIALS_ID) {
-		dockerHelper.dockerBuildAndPushImage(archImageName, config.dockerBuildArgs ?: '.')
+		String args = config.dockerBuildArgs ?: '.'
+		args = '--network host ' + args
```

Comment:
is `--network host .` valid ?

Reply:
it will run as ``docker build -t $archImageName --network host .``

Reply:
i was more asking if '.' was valid, since it seems to not be added when there are args? or is it intended to be part of args

Reply:
yea ``docker build`` takes a path for its current working directory.  The default is '.' but if a project override with docker args then the Jenkinsfile need to also add this path because it can be different.